src/DacpacTool/Extensions.cs

```diff
@@ -72,6 +72,30 @@ public static string Format(this BatchParserExecutionErrorEventArgs args, string
             return outputMessageBuilder.ToString();
         }
 
+        public static string Format(this DacModelException exception, string fileName)
+        {
+            var stringBuilder = new StringBuilder();
+
+            foreach (var modelError in exception.Messages)
+            {
+                stringBuilder.Append(fileName);
+                stringBuilder.Append('(');
```

Comment:
Is the `(1,1)` here just to fit with the formatting of other errors? Is it required to be in that format? If it's not required, I'd personally skip it, since in my opinion it may cause people to look in the wrong place for an error.

Reply:
This is to help errors display properly in the VS error list. The convention is to use 1,1 is the exact file location is not available.

Reply:
Confirmed the 1,1 convention

Reply:
Thanks for clarifying, I didn't realize that was a convention 👍